cfn-resources/private-endpoint-service/cmd/resource/resource.go

```diff
@@ -0,0 +1,288 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resource
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	resource_constats "github.com/mongodb/mongodbatlas-cloudformation-resources/private-endpoint-service/cmd/constants"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/private-endpoint-service/cmd/resource/steps/privateendpointservice"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/profile"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	"go.mongodb.org/atlas/mongodbatlas"
```

Comment:
is there a reason we are not making use of the new go SDK in the private-endpoint-service?

Reply:
it was just for timing issues, this resource already existed (this is the former private-endpoint) so since there was a task for this change, i just removed the private endpoint, and the aws private endpoint from this one